security_guidelines.md

```diff
@@ -0,0 +1,52 @@
+# Security Guidelines
+Security guidelines to jumpstart implementations and avoid security pitfalls.
+
+## Certificates
+● Purchase certificate solely for OpenGDPR to prevent issues elsewhere with your infrastructure if
```

Comment:
There's some formatting issues with the markdown here and below. Rather than using "●" you should just use regular dashes. See the preview here: https://github.com/opengdpr/opengdpr/blob/0f6d0f1ce5dca3a23be4bf13480ff5910a7bfb93/security_guidelines.md